### PR TITLE
Flatten Greece and Add Support for three questions flow

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello card](https://trello.com/c/WysStRHr/580-5-marriage-abroad-improvement-greece-6-outcomes)

## Description 

The goal of this PR is to migrate the flow for Greece to use the outcome per path strategy and also add support for three questions flow.

As you will  observe (in the first commit ) the title only contains opposite sex title, this
because the same sex title doesn't use the content title block.  It uses the markdown syntax, this will be fix in a7cfd83.

## Factcheck

[Preview link](https://smart-answers-pr-2977.herokuapp.com/marriage-abroad/y/greece/uk)
[GOVUK](https://www.gov.uk/marriage-abroad/y/greece/uk/)

## Expected changes

- Flatten content for Greece
- Enabling outcome per path strategy
- Removed partner nationality question for Greece
- Enabling  three questions flow

### Before 

screen shot 2017-03-27 at 16 52 00

### After

screen shot 2017-03-27 at 16 53 53

### CODE REVIEW

**Comment by leenagupte** [comment]
@ikennaokpala It might be useful to the deployer to highlight which question is being removed from the flow for Greece.

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/greece/_title.govspeak.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <% if calculator.partner_is_opposite_sex? %>
   Marriage in Greece
```
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/greece/_title.govspeak.erb]: The means need to fix it in the commit `Fix same sex title discrepancy`

**Comment by leenagupte** [comment]
Thanks for making the changes @ikennaokpala.

**Comment by ikennaokpala** [comment]
Rebasing from 2cbd8e6bb952325859a6696d18110a7194a34ffa to 9fed27130e2c8b80f4fdd48f81fcc519a77fb959

**Comment by ikennaokpala** [comment]
Closing this as it has been superseded by #3094 and #3092

**Pull request closed without merging**